components/Dialogs/src/MDCAlertController.m

```diff
@@ -227,6 +227,10 @@ - (void)loadView {
   self.alertView = (MDCAlertControllerView *)self.view;
 }
 
+- (MDCAlertControllerView *)alertControllerView {
+  return (MDCAlertControllerView *)self.view;
```

Comment:
Return `self.alertView` instead so that you don't have to cast.

Reply:
Ah, then we should document that we're depending on that side effect behavior.

```
[self loadViewIfNeeded];
return self.alertView;
```

Reply:
Done.